backends/cuda/ceed-cuda-qfunction-load.cpp

```diff
@@ -66,9 +46,17 @@ extern "C" int CeedCudaBuildQFunction(CeedQFunction qf) {
   ierr = CeedQFunctionGetFields(qf, &numinputfields, &qfinputfields, &numoutputfields, &qfoutputfields);
   CeedChkBackend(ierr);
 
+  // Additional kernels
+  char source_path[CEED_MAX_PATH_LEN] = __FILE__;
+  CeedInt end = strrchr(source_path, '/') - source_path;
+  strncpy(&source_path[end], "/kernels/cuda-qfunction.h", 26);
+  char *qFunctionKernels;
+  ierr = CeedLoadSourceToBuffer(ceed, source_path, &qFunctionKernels);
```

Comment:
It seems that this pattern appears in multiple places, this should be factorized into a function I think.